pkg/processor/trigger/http/trigger.go

```diff
@@ -565,6 +565,16 @@ func (h *http) handleRequest(ctx *fasthttp.RequestCtx) {
 	case string:
 		ctx.WriteString(typedResponse) // nolint: errcheck
 	}
+
+	h.setFooHeaderIfNeeded(ctx)
+}
+
+func (h *http) setFooHeaderIfNeeded(ctx *fasthttp.RequestCtx) {
+	if h.configuration.RuntimeConfiguration.Spec.Foo != nil {
+		ctx.Response.Header.Set("X-Nuclio-Foo", *h.configuration.RuntimeConfiguration.Spec.Foo)
+	} else if h.configuration.RuntimeConfiguration.PlatformConfig.Foo != nil {
+		ctx.Response.Header.Set("X-Nuclio-Foo", *h.configuration.RuntimeConfiguration.PlatformConfig.Foo)
+	}
```

Comment:
While we do have access to the platform config from the trigger, it is better to the the enrichment on the dashboard side, when creating the function. The processor should get the configuration after all of the enrichments.
You can look at the plaform's `enrichAndValidateFunctionConfig` - this is where the function config is enriched with default values.